### ISSUE TITLE
Redundancy of "was: Untitled Wekan board" and identical titles

### ISSUE THREAD

**Issue by neynah** [sandstorm-io/sandstorm#2520]
When the previous grain title is "Untitled Wekan board" or [the same grain title as it currently is] I think displaying "was:" text here is redundant and adds no value. I think we should not display previous titles in cases where the title is "Untitled..." or the same title as it currently is. Curious as to what other's think.
![image](https://cloud.githubusercontent.com/assets/855341/18398148/474cf394-7680-11e6-93b2-06fac139c072.png)

**Comment by kentonv** [comment]
I agree that if the titles match exactly the "was" shouldn't be displayed -- but I thought it did that already, so that seems to be a bug.

Regarding "Untitled...", what we should really do is prevent people from sharing a grain that starts with "Untitled...". Google Docs does this: it prompts you to choose a title before you share.

**Comment by neynah** [comment]
>Regarding "Untitled...", what we should really do is prevent people from sharing a grain that starts with "Untitled...". Google Docs does this: it prompts you to choose a title before you share.

Yes, we should do this!

**Comment by dwrensha** [comment]
> I agree that if the titles match exactly the "was" shouldn't be displayed -- but I thought it did that already, so that seems to be a bug.

One way something like this can happen is:

  1. Alice shares grain A with Title "first title" to Bob
  2. Bob changes the title to "second title"
  3. Alice changes the title to "second title"
  4. Bob now sees the grain as "second title (renamed from: second title)"
